Pipfile.lock

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
why re pipfile/pipfile.lock

Reply:
setup.py and requirements.txt seems sufficient, rather than duplicating requirements.

Reply:
This was to provide the same ergonomics as `poetry`, where we can `pipenv run ...` and `pip env lock` our dev dependencies and have the whole chain locked so we don't get random upgrades during CI/CD that break anything. 

I used the `Pipfile` to generate the `requirements-dev.txt` file, so it isn't a duplicate, just included for convience.

I'm not tied to `pipenv` or anything like that.   This is a library so its not going to have strict dependency requirements, I was just trying to match the experience we had with `poetry`

Reply:
I removed pipenv